web-common/src/features/project/deploy-errors.ts

```diff
@@ -41,7 +41,10 @@ export type DeployError = {
   message: string;
 };
 
-export function extractDeployError(error: ConnectError): DeployError {
+export function extractDeployError(
+  error: ConnectError,
+  orgOnTrial: boolean,
```

Comment:
This feels like a random parameter to add to this function – because it doesn't relate to "extracting" information from the error message. Maybe it'd make sense with a small name change: `makeDeployError`, `makePrettyDeployError`?